src/plugins/management/public/plugin.ts

```diff
@@ -186,18 +207,24 @@ export class ManagementPlugin implements Plugin<ManagementSetup, ManagementStart
       workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
       mount: async (params: AppMountParameters) => {
         const { renderApp } = await import('./landing_page_application');
-        const [coreStart] = await core.getStartServices();
+        const [coreStart, { navigation }] = await core.getStartServices();
         const navLinks = (
           await getNavLinksByNavGroupId(DEFAULT_NAV_GROUPS.settingsAndSetup.id)
         ).filter((navLink) => navLink.id !== settingsLandingPageId);
 
+        coreStart.chrome.setBreadcrumbs([
```

Comment:
Nit: How about move page title update logic to FeatureCards component?

Reply:
I would say that I have no preference whether to still maintain it here or move it into component, because I don't think it will have much impact on it or maintainability.

Reply:
I would say that I have no preference whether to still maintain it here or move it into component, because I think there won't be much difference on it or maintainability.